<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="2">
    <spacer name="VerticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="LanguageLabel">
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="QuickSearchCaseCheckbox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Quick filter case sensitive</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="LanguageCombobox">
     <item>
      <property name="text">
       <string>English</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>French</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="ButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="ThemeCombobox">
     <item>
      <property name="text">
       <string>Classic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dark</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OLED</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Midnight Purple</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="ThemeLabel">
     <property name="text">
      <string>Theme</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="DateFormatLabel">
     <property name="text">
      <string>Date format</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="DateFormatCombobox">
     <item>
      <property name="text">
       <string>yyyy-MM-dd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dd/MM/yyyy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MM/dd/yyyy</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="MatrixModeCheckbox">
     <property name="toolTip">
      <string>There is no spoon.</string>
     </property>
     <property name="text">
      <string>Matrix mode</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="MoreLogsCheckbox">
     <property name="toolTip">
      <string>Show more logs if launched from a terminal (for debugging purpose)</string>
     </property>
     <property name="text">
      <string>More logs</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>LanguageCombobox</tabstop>
  <tabstop>ThemeCombobox</tabstop>
  <tabstop>DateFormatCombobox</tabstop>
  <tabstop>MatrixModeCheckbox</tabstop>
  <tabstop>QuickSearchCaseCheckbox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
